packages/keycloak-user-management/src/components/index.tsx

```diff
@@ -0,0 +1,261 @@
+import * as React from 'react';
+import { notification, Row, Col, Button, Space, Table, Divider, Popconfirm } from 'antd';
+import { KeycloakService } from '@opensrp/keycloak-service';
+import { Link } from 'react-router-dom';
+import { history } from '@onaio/connected-reducer-registry';
+import Ripple from '../components/Loading';
+import HeaderBreadCrumb from './HeaderBreadCrumb';
+import {
+  KeycloakUser,
+  fetchKeycloakUsers,
+  getKeycloakUsersArray,
+  removeKeycloakUsers,
+  reducerName as keycloakUsersReducerName,
+  reducer as keycloakUsersReducer,
+  getAccessToken,
+} from '@opensrp/store';
+import { Store } from 'redux';
+import { PropsTypes } from './CreateEditUser';
+import { connect } from 'react-redux';
+import { Dictionary } from '@onaio/utils';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+
+reducerRegistry.register(keycloakUsersReducerName, keycloakUsersReducer);
+
+// const { Content } = Layout;
+
+// props interface to Admin component
+export interface Props {
+  serviceClass: typeof KeycloakService;
+  fetchKeycloakUsersCreator: typeof fetchKeycloakUsers;
+  removeKeycloakUsersCreator: typeof removeKeycloakUsers;
+  keycloakUsers: KeycloakUser[];
+  accessToken: string;
+}
+
+/** default props for UserIdSelect component */
+export const defaultProps = {
+  accessToken: 'hunter 2',
+  serviceClass: KeycloakService,
+  fetchKeycloakUsersCreator: fetchKeycloakUsers,
+  removeKeycloakUsersCreator: removeKeycloakUsers,
+  keycloakUsers: [],
+};
+
+interface TableData {
+  key: number | string;
+  id: string | undefined;
+  username: string | undefined;
+  email: string | undefined;
+  firstName: string | undefined;
+  lastName: string | undefined;
+}
+
+/**
+ * Handle user deletion
+ */
+export const deleteUser = (props: Props, userId: string): void => {
+  const {
+    serviceClass,
+    fetchKeycloakUsersCreator,
+    removeKeycloakUsersCreator,
+    accessToken,
+  } = props;
+  const serviceDelete = new serviceClass(
+    accessToken,
+    `/users/${userId}`,
+    'https://keycloak-stage.smartregister.org/auth/admin/realms/opensrp-web-stage'
+  );
+  const serviceGet = new serviceClass(
+    accessToken,
+    '/users',
+    'https://keycloak-stage.smartregister.org/auth/admin/realms/opensrp-web-stage'
```

Comment:
- these should be props
- please no magic strings

Reply:
Theres an ongoing refactor to have the`baseURL` passed as props to the user management package by @kelvin-muchiri

Reply:
I would like us to have an issue that explicitly states this

Reply:
https://github.com/OpenSRP/web/issues/62